tests/assets/a1.json

```diff
@@ -11,7 +11,7 @@
             "en-US":"created"
         }
     },
-    "object":{
+    "statementobject":{
```

Comment:
Again, I think all these references in the assets need to be changed back.
```suggestion
    "object":{
```

Reply:
see #18